src/Compilers/Core/MSBuildTask/ManagedCompiler.cs

```diff
@@ -845,7 +851,15 @@ private void AddAdditionalFilesToCommandLine(CommandLineBuilderExtension command
         /// </summary>
         private void AddEmbeddedFilesToCommandLine(CommandLineBuilderExtension commandLine)
         {
-            if (EmbeddedFiles != null)
+            // If EmbedAllSources is specified we can ignore files explicitly specified in EmbeddedFiles.
+            // They do not contribute to the set of files included in the compilation. 
+            // They only designate which files that are being compiled should be also embedded.
```

Comment:
How do I say "embed all sources and this one extra file like a XAML file or something?" not possible?

Reply:
Actually, I was mistaken. Fixing.

Reply:
Fixed.